EdxMobile/app/navigators/AppNavigator.tsx

```diff
@@ -18,10 +18,17 @@ import Config from "../config"
 import { useStores } from "../models"
 import {
   LoginScreen,
-  WelcomeScreen,
+  WelcomeScreen
 } from "../screens"
+import CourseDetailScreen from "../screens/CourseDetailScreen"
```

Comment:
This needs to be added to the index file then added to the list above

Reply:
This file? https://github.com/jonmahoney15/edx-mobile/blob/main/EdxMobile/index.js

Reply:
no the index.js within the screens directory